sql/branch.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TYPE payment_net ADD VALUE 'samurai';
+ALTER TYPE payment_net ADD VALUE 'stripe';
+
+ALTER TABLE exchanges ADD UNIQUE (ref);
```

Comment:
Oh wait - `network` is in the exchange routes table. Hmm, how would this work out? Isn't it possible that the `ref` from a `braintree` and that from a `balanced` payment collide?